com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -75,6 +75,11 @@ public class NetworkSceneManager
         /// </summary>
         public event NotifyClientAllClientsLoadedSceneDelegate OnNotifyClientAllClientsLoadedScene;
 
+        // This is a temporary solution for switching scenes during unit tests using NetworkSceneManager
+        // When the NetworkSceneManager is overhauled, this will probably no longer be needed.
+        internal delegate AsyncOperation OverrideLoadSceneAsyncDelegateHandler(string targetscene, LoadSceneMode scenemode);
+        internal OverrideLoadSceneAsyncDelegateHandler OverrideLoadSceneAsync;
```

Comment:
I do not think this workaround/override is necessary, even at this stage.
I saw the other draft PR you're preparing and we could simply use short form of scene name instead of fullname and it'd just work fine.
this PR is not a refactor, it's a feature (`feat:`) and I'm not sure if we really need it.

Reply:
You gave me an idea... :)